apps/frontend/public/static/export/template.html

```diff
@@ -0,0 +1,284 @@
+    <style type="text/css">
+      .results-set {
+      min-width: 40vh;
+      padding-bottom: 10px;
+      }
+      .justify-center, .apexcharts-canvas {
+      display: block;
+      margin: auto;
+      justify-content: center;
+      align-items: center;
+      text-align: center;
+      }
+      .compliance-card {
+      min-height: 300px;
+      }
+      .apexcharts-ycrosshairs {
+      display: none;
+      }
+      .apexcharts-tooltip-series-group .apexcharts-active {
+      display: none;
+      }
+      @media (min-width: 1101px) and (max-width: 1446px) {
+      .tall-card {
+      height: 200px;
+      }
+      }
+      @media (min-width: 1446px) and (max-width: 1886px) {
+      .tall-card {
+      height: 190px;
+      }
+      }
+      @media (min-width: 1886px) {
+      .tall-card {
+      height: 130px;
+      }
+      }
+    </style>
+    <style type="text/css" media="print">
+      @page { size: landscape; }
+    </style>
+    <script>
+      /** Saves the current page **/
+      function savePage(){
+        const pageBlob = new Blob([this.s2ab(document.documentElement.outerHTML)], {type: 'application/html'})
+        saveFile(pageBlob, `{{exportType}}_Report_${new Date().toString()}.html`.replace(/[ :]/g, '_'))
+      }
+      
+      /** Downloads a blob as a specified filename **/
+      function saveFile(blob, filename) {
+        if (window.navigator.msSaveOrOpenBlob) {
+          window.navigator.msSaveOrOpenBlob(blob, filename);
+        } else {
+          const a = document.createElement('a');
+          document.body.appendChild(a);
+          const url = window.URL.createObjectURL(blob);
+          a.href = url;
+          a.download = filename;
+          a.click();
+          setTimeout(() => {
+            window.URL.revokeObjectURL(url);
+            document.body.removeChild(a);
+          }, 0)
+        }
+      }
+      
+      /** Converts a string to an array buffer */
+      function s2ab(s) {
+        const buf = new ArrayBuffer(s.length); //convert s to arrayBuffer
+        const view = new Uint8Array(buf); //create uint8array as viewer
+        for (let i = 0; i < s.length; i++) {
+          view[i] = s.charCodeAt(i) & 0xff; //convert to octet
+        }
+        return buf;
+      }
+    </script>
+  </head>
+  <body>
+    <nav class="navbar fixed-top navbar-expand-lg navbar-light bg-light">
+      <div class="container-fluid">
+        <a class="navbar-brand" href="#">Heimdall Report</a>
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation"> <span class="navbar-toggler-icon"></span> </button>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+          <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+            <li class="nav-item"> <a class="nav-link active" aria-current="page" href="#">Profile Info</a> </li>
```

Comment:
I'm not sure what the intent of this link is.

Reply:
This was intended to jump to the top alongside the ability to jump between control sets, however, the library to do this seems to be causing the copied Apex charts to disappear.
![image](https://user-images.githubusercontent.com/66680985/121593844-5507f280-ca0a-11eb-82ce-42f1f5332016.png)